ci/build-wheel-deps.sh

```diff
@@ -1,28 +1,31 @@
 #!/bin/bash
 set -e -x
 
-PACKAGES=(
-    zlib-1.2.11
-    libpng-1.6.37
-    freetype-2.10.4
-    harfbuzz-2.6.4
-)
+curl -L "https://downloads.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1.6.37.tar.gz" -o libpng-1.6.37.tar.gz
+tar zxf libpng-1.6.37.tar.gz
+pushd libpng-1.6.37
+./configure
+make -j $(nproc)
+make install
+popd
 
-URLS=(
-    https://www.zlib.net/zlib-1.2.11.tar.gz
-    https://downloads.sourceforge.net/project/libpng/libpng16/1.6.37/libpng-1.6.37.tar.gz
-    https://download.savannah.gnu.org/releases/freetype/freetype-2.10.4.tar.gz
-    https://github.com/harfbuzz/harfbuzz/archive/2.6.4.tar.gz
-)
+curl -L "https://download.savannah.gnu.org/releases/freetype/freetype-2.12.1.tar.gz" -o freetype-2.12.1.tar.gz
+tar zxf freetype-2.12.1.tar.gz
+pushd freetype-2.12.1
+./configure
+make -j $(nproc)
+make install
+popd
 
-for idx in {0..3}; do
-    PACK=${PACKAGES[$idx]}
-    URL=${URLS[$idx]}
-    curl -L "${URL}" -o ${PACK}.tar.gz
-    tar zxf ${PACK}.tar.gz
-    pushd ${PACK}
-    ./configure
-    make
-    make install
-    popd
-done
+curl -L "https://github.com/harfbuzz/harfbuzz/archive/refs/tags/2.6.8.tar.gz" -o harfbuzz-2.6.8.tar.gz
+tar zxf harfbuzz-2.6.8.tar.gz
+pushd harfbuzz-2.6.8
+
+# missing macro in the locally installed automake
+cp /usr/share/aclocal/pkg.m4 /usr/local/share/aclocal
```

Comment:
Should this be relayed upstream to `RalfG/python-wheels-manylinux-build`?

Reply:
This seems to be a known issue in the base manylinux image: https://github.com/pypa/manylinux/issues/731

Reply:
2020... 😬